community/py-sqlparse/APKBUILD

```diff
@@ -2,22 +2,28 @@
 # Maintainer: Thomas Boerger <thomas@webhippie.de>
 pkgname=py-sqlparse
 _pkgname=sqlparse
-pkgver=0.2.2
-pkgrel=2
+pkgver=0.2.4
+pkgrel=0
 pkgdesc="A non-validating SQL parser module for Python"
 url="https://sqlparse.readthedocs.io"
 arch="noarch"
 license="BSD"
 depends=""
-makedepends="python2-dev py-setuptools python3-dev"
+makedepends="python2-dev py-setuptools pytest python3-dev"
```

Comment:
The `pytest` should be in `checkdepends` kinda https://github.com/alpinelinux/aports/pull/5476

Reply:
Thanks, I've changed it!